framework/rediscache.py

```diff
@@ -52,6 +52,9 @@ def set(key, value, time=86400):
   if redis_client is None:
     return
 
+  if value is None:
```

Comment:
Why not allow setting a value of None?

I think None would work: calling `pickle.dumps(None)` gives a binary string, and `pickle.loads(pickle.dumps(None))` gives None.    It could be useful to cache a None in some case, although I don't think we do that currently.

Reply:
Yea None indeed works. There were testing errors and I suspected it was due to None. Maybe unrelated

Reply:
Okie I just remembered.  `pickle.loads(None)` in `get` throws errors when a `key` doesn't exist